manifests/init.pp

```diff
@@ -0,0 +1,41 @@
+# @summary
+#   Creates a xfs_quota bash script in /etc/telegraf/scripts, and a conf file to run it
+#
+# @param enable
+#   Enables the xfs quota checks
+# 
+# @param quota_path
+#   Specifies the path of the xfs volume
+#
+# @example
+#   include profile_xfs_quota
+class profile_xfs_quota (
+  Boolean $enable,
+  String $quota_path,
```

Comment:
Longer term this should probably support more than one volume path to be able to check quotas for.

If that slows you down too much now, feel free to leave as is, but create an issue for this feature in this repo for future enhancements.

Reply:
As I look at this more, it sounds like https://git.ncsa.illinois.edu/ici-monitoring/ici-developed-checks/ may not support a list of paths to check. 

If that is the case, again feel free to ignore my suggestion. But I think it would be good to create an issue to support multiple paths in https://git.ncsa.illinois.edu/ici-monitoring/ici-developed-checks/ .

Reply:
I've created the issue here https://git.ncsa.illinois.edu/ici-monitoring/ici-developed-checks/-/issues/1